src/tests/util.rs

```diff
@@ -130,10 +135,14 @@ pub trait RequestHelper {
         Response::new(self.app().0.middle.call(&mut request))
     }
 
+    /// Search for crates matching a query string
+    fn search(&self, query: &str) -> CrateList {
+        self.get_with_query("/api/v1/crates", query).good()
+    }
+
     /// Get the crates owned by the specified user.
-    fn crates_owned_by_user_id(&self, id: i32) -> CrateList {
-        let query = format!("user_id={}", id);
-        self.get_with_query("/api/v1/crates", &query).good()
+    fn search_by_user_id(&self, id: i32) -> CrateList {
```

Comment:
this is a much better name, because this action is implemented using search. it's a good reminder rather than leading the reader to believe it's completely different 👍